project/frontend/src/components/ProfileForm.js

```diff
@@ -0,0 +1,178 @@
+import './PreferenceForm.css';
+
+import React, { useState, useEffect, useContext } from 'react';
+
+import CuisineAutocomplete from './CuisineAutocomplete.js';
+import { Slider } from 'rsuite';
+import { useHistory } from 'react-router-dom';
+import { AuthContext } from './Authentication.js';
+
+import Place from '../assets/place.svg';
+import Cuisine from '../assets/cuisine.svg';
+import Distance from '../assets/distance.svg';
+import Experience from '../assets/food_service.svg';
+import Price from '../assets/dollar.svg';
+
+/**
+ * A form used to gather the restaurant preferences of a user for the 'Find My Match'
+ * service.
+ *
```

Comment:
er... a lot of this code is the same as the preference form component: https://github.com/googleinterns/step127-2020/blob/master/project/frontend/src/components/PreferenceForm.js ? is that intentional or can it be shared using props ?

Reply:
Yup right now it's copied but I don't plan on keeping it like this!! copied from my comment upon opening the PR: "opening this draft because i want opinions about how i should make components from the preference form page so that it can be used on the profile page." this morning Kira, Alex, and I were discussing how to split up the components in the preference form page so that they can be used here.

I wanted to show them what code would be necessary to include in the profile page to help brainstorm how we could create components that would work for both the preference form and the profile page.

Reply:
ah ok, sounds good! sorry for hopping on preemptively :D